### PR TITLE
Fixed Merge Issue: Added TodoItem Model

### PR DESCRIPTION
**Pull request by adigulalkari**
I have fixed the merge conflicts.

### CODE REVIEW

File: app/models.py [hunk 1]
```diff
@@ -1,27 +1,22 @@
 """ Module defining all database models"""
 
 # Imports
+from xmlrpc.client import Boolean
```
Comment (Dhruv9449) [inline app/models.py]: Delete this import

File: app/models.py [hunk 2]
```diff
@@ -1,27 +1,22 @@
...
 from sqlalchemy import Column, Integer, String, TIMESTAMP
```
Comment (Dhruv9449) [inline app/models.py]: Import boolean from from sqlalchemy

File: app/models.py [hunk 3]
```diff
@@ -1,27 +1,22 @@
...
-    - completed: Boolean value indicating if the todo item is completed or not
-    """
-    
     __tablename__ = "todo_items"
     
     id = Column(Integer, primary_key=True, index=True)
-
+    title = Column(String)
+    description = Column(String(length = 300))
+    created_at = Column(TIMESTAMP)
```
Comment (Dhruv9449) [inline app/models.py]: add default value of created_at and updated_at and set it to now.

File: app/models.py [hunk 4]
```diff
@@ -1,27 +1,22 @@
...
+    updated_at = Column(TIMESTAMP)
+    completed = Column(Boolean)
 
     def __repr__(self):
         """
```
Comment (Dhruv9449) [inline app/models.py]: Add this

**Review by Dhruv9449 — CHANGES_REQUESTED** [review]

**Comment by adigulalkari** [comment]
Thanks for the feedback.. can u please check if everything is fine now that i have changed the requested changes.
✨

**Pull request closed without merging**